internal/models/source.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type Source struct {
+	ID        uint64 `db:"id"`
+	SourceID  string `db:"source_id"`
+	Name      string `db:"name"`
+	OrgID     string `db:"org_id"`
+	AuthID    string `db:"auth_id"`
+	CreatedAt string `db:"created_at"`
+}
```

Comment:
So I am already confused what source_id and auth_id is at this point, I suggest that from now on we will document all models pretty extensively. I suggest to have documentation for the model itself (what it represents) as well as for each individual columns. @adiabramovitch @ezr-ondrej @amirfefer

Reply:
I'm totally up for it, is there some standards for documenting Go code inline?